### DISCUSSION TITLE
How to add a gist to a repository

### DISCUSSION THREAD

**Discussion by dragontail281** [community/community#21835] [New to GitHub]
<p>I added a gist before creating my repository.  </p>
<p>Is there a way to move it to the repository or should I just build it there also?</p>
<p>Thanks</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! There is not a way to transfer Gist to a specific repo. However, given gists are just rendered files (in markdown, HTML, etc.) you can just copy the file content and then manually move it to the repo of your choice. GitHub should render the file the same in the repo as it did on the Gists page. </p>

**Reply by vbrozik** [reply]
Unfortunately this way you lose your commit history of the Gist :(

If you do not have the new repository yet, you can create it from the Gist:
https://gist.github.com/ishu3101/830b556b487de5d69690

Or you can merge the Gist repository into another repository:
https://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories
